packages/quantic/force-app/examples/main/lwc/exampleQuanticRecommendationList/exampleQuanticRecommendationList.html

```diff
@@ -0,0 +1,26 @@
+<template>
+  <c-example-layout
+    title={pageTitle}
+    description={pageDescription}
+    show-preview={isConfigured}
+    expected-events={expectedEvents}
+  >
+    <div slot="configuration">
+      <c-configurator options={options} ontryitnow={handleTryItNow}>
+      </c-configurator>
+    </div>
+
+    <c-quantic-recommendation-interface slot="preview" engine-id={engineId}>
+      <c-quantic-recommendation-list
+        engine-id={engineId}
+        recommendation={config.recommendation}
+        number-of-recommendations={config.numberOfRecommendations}
+        number-of-recommendations-per-row={config.numberOfRecommendationsPerRow}
```

Comment:
Should we have this instead, would this be less confusing?

```suggestion
        number-of-recommendations={config.numberOfRecommendations}
        recommendations-per-row={config.recommendationsPerRow}
```

Reply:
sure!